### ISSUE TITLE
Submit BugZilla changes to upstream?

### ISSUE THREAD

**Issue by rugk** [jcsteh/axSGrease#1]
What about submitting the Bugzilla changes to Mozilla? As Bugzilla is open-source I think more people can profit from this "patch".

**Comment by jcsteh** [comment]
I'm not familiar with the Bugzilla source code and I don't have time to set up a Bugzilla server instance to work on this. However, I filed [Mozilla bug 583087](https://bugzilla.mozilla.org/show_bug.cgi?id=583087) regarding adding appropriate headings. (The lack of action on this bug is why I developed the Greasemonkey script.)

Note that the Bugzilla user interface is currently being redesigned, so we're hoping to have these issues addressed in the new UI before it goes live. Relevant bugs have been filed; here are [all Bugzilla accessibility bugs](https://bugzilla.mozilla.org/buglist.cgi?quicksearch=product%3Abugzilla%20keywords%3Aaccess&list_id=13081300).
